blogposts/remote-software-development-made-easier.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Remote software development made easier"
+author: Sourcegraph
+authorUrl: https://twitter.com/srcgraph
+publishDate: 2020-03-22T17:59
+tags: [blog]
+slug: remote-software-development-made-easier
+heroImage: /blog/remote-easier.jpg
+published: true
+---
+
+Sourcegraph has been remote-friendly from the beginning — our two longest-tenured teammates have worked remotely since 2015 from Arizona and South Africa! We moved to remote-first in May of 2019 and completed our transition to all-remote with the ringing-in of 2020.
+
+We have compiled some excellent tips and tricks from fellow [teammates](https://about.sourcegraph.com/company/team) on how to live your best distributed work life!
+
+## Tips for working remotely
+
+### Home office and desk setup
+
+- **Separate work and home:** “Try working in a different space to where you relax/sleep.”
```

Comment:
Any specific reason why all the bullet points have their descriptions in quotes? The [Sourcegraph style guide](https://about.sourcegraph.com/handbook/communication/style_guide) recommends "\[punctuation] goes outside of quotation marks" which this breaks. Also, personally,

- **Separate work and home:** Try working in a different space to where you relax/sleep.

Is more readable than

- **Separate work and home:** “Try working in a different space to where you relax/sleep.”

primarily because quotes have a well defined context (you're quoting someone), just like you're quoting Michael and Quinn in the later part of this post. I'd recommend getting rid of them, but I'd love to hear if you chose typestyle this for a particular reason that I'm missing.

Reply:
The reason is that we are quoting our teammates. As mentioned in the paragraph above: 
```We have compiled some excellent tips and tricks from fellow [teammates](https://about.sourcegraph.com/company/team) on how to live your best distributed work life!```
And regarding punctuation inside the quotes, our style guide states:
```Punctuation goes outside of quotation marks, except in marketing when displaying a quote.``` 
In this case, these are quotes.

Reply:
I found this a bit weird as well, but love that we're using quotes from our teammates. 

@vanesa What about reusing quotation format from our testimonials styles, as that would make it obvious more clear that we're using quotes from teammates, and would make the blog post more visual.

Reply:
Thanks, @vanesa! That makes a lot of sense, I missed that. Maybe then we can cite the quoter? Something like Separate work and home: “Try working in a different space to where you relax/sleep.” – Example name? Only because it's not necessarily clear that it's a quote without the citation; if we remove the quotation marks, it still works as independent points. You're the best judge!

Reply:
@ryan-blunden yes I'm currently reformatting this post into the format with have for case studies also due to other reasons. I'll see how it looks if we remove the quotes.
@AnandChowdhary we tried that in the past with our pdf version and it just didn't work out because many of the quotes have multiple authors and it just looks very cluttered. Will see how the removal of the quotes work in the new format. Stay tuned 😄.